app/work.go

```diff
@@ -0,0 +1,37 @@
+package app
+
+import "github.com/ubclaunchpad/cumulus/blockchain"
+
+const (
+	// BlockQueueBuffer is the size of the BlockQueue channel.
+	BlockQueueBuffer = 100
+	// TransactionQueueBuffer is the size of the BlockQueue channel.
+	TransactionQueueBuffer = 100
+)
+
+// Responder is used to handle requests who require a response.
+type Responder interface {
+	Send(ok bool)
+	Lock()
```

Comment:
`Send` is sort of prototype-y. Not sure what message may need to go back to the app.